include/dsAudio.h

```diff
@@ -1122,7 +1053,8 @@ dsError_t  dsGetAudioLevel(intptr_t handle, float *level);
 /**
  * @brief Sets the audio volume level of an audio port.
  * 
- * This function sets the audio volume level to be used on the audio port corresponding to specified port handle.
+ * For sink devices, this function sets the audio volume level to be used on the audio port corresponding to specified port handle.
```

Comment:
Please update properly based on existing product HAL implementation

Reply:
Please find the updated description below:

**For sink devices, this function sets the audio volume level to be used for Speaker(dsAUDIOPORT_TYPE_SPEAKER) and Headphone(dsAUDIOPORT_TYPE_HEADPHONE) ports.**

Reply:
Looks ok